### PR TITLE
Adding list support

### PR DESCRIPTION
**Pull request by drusellers**
Initial list support for getting all issues

### CODE REVIEW

File: hucache.http/Cache.fs [hunk 1]
```diff
@@ -4,11 +4,16 @@ open hucache.http.caches.postgres
...
     | None -> 
         let issue = fetchIssue key
         match issue with 
         | Some i -> Some((storeIssue (key, i)))
         | None -> None
+
+let lookupIssues (key: RepoKey) : FullPayload list = 
+    match loadIssues key with 
+    | [] -> []
+    | i -> i
```
Comment (dahlbyk) [inline hucache.http/Cache.fs:19]: Is there more to be done here eventually? Otherwise you could just `let lookupIssues = loadIssues` for now.

**Comment by dahlbyk** [comment]
If we're going to be doing a fair amount of manipulating `ExpandoObject`, it may be worth pulling in http://www.nuget.org/packages/ImpromptuInterface.FSharp/ (and more generally, https://github.com/ekonbenefits/impromptu-interface/blob/0621eeb1a5c7b0c7b5d24f5acc9f05e2d3892ca2/ImpromptuInterface.FSharp/FSharp.fs is an interesting read).

File: hucache.http/caches/Postgres.fs [hunk 2]
```diff
@@ -61,7 +61,25 @@ let loadIssue (key:IssueKey) : FullPayload option =
...
+    pp.Add("repo", key.repo)
+              
+    conn.Query<R>("SELECT headers AS Headers, payload AS Payload 
+                   FROM github.issues
+                   WHERE owner = @owner
+                   AND repo = @repo", p)
+    |> Seq.map (fun r ->
+            let h = deserialize r.Headers
+            let p = r.Payload
+            {headers=h; payload=p}
```
Comment (dahlbyk) [inline hucache.http/caches/Postgres.fs]: Name this function to reuse above (`Seq.tryPick (myFunc >> Some)`)?

File: hucache.http/Cache.fs (hunk 1 above)
Comment (drusellers) [inline hucache.http/Cache.fs:19]: yeah, i was thinking we _may_ want to kick off some kind of update maybe?

**Comment by drusellers** [comment]
> If we're going to be doing a fair amount of manipulating ExpandoObject

No, idea. Since we don't have a lot of code yet, not a lot to do here. I'm ok with it as is. Once the next level of NPGSQL comes in with a little bit better hstore support we should be even better here.

File: hucache.http/caches/Postgres.fs (hunk 2 above)
Comment (drusellers) [inline hucache.http/caches/Postgres.fs]: fixed

**Pull request merged**